### PR TITLE
Registration ui

### PR DESCRIPTION
**Pull request by BorysYudin**

### CODE REVIEW

File: events/static/js/app/registration/float-label-field.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import Rules from "./rules";
+import TextField from 'material-ui/TextField';
+import Popover from 'material-ui/Popover/Popover';
+import validator from 'validator';
+var ReactBootstrap = require('react-bootstrap');
```
Comment (Scandie) [inline events/static/js/app/registration/float-label-field.js]: @BorysYudin delete useless ReactBootstrap import.

File: events/static/js/app/registration/form.js [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+        let registration_data = Object.assign({}, this.state);
+        delete registration_data.password_confirm;
+        delete registration_data.formIsValid;
+        let date = new Date(registration_data["birth_date"]);
+        registration_data["birth_date"] = new Date(date.getMonth() + "/" +
+                                                   date.getDate() + "/" +
+                                                   date.getFullYear()).getTime() / MILLISECONDS;
+
+        axios.post(this.props.url, registration_data)
+            .then((responce) => this.props.handleSubmit(responce.data.message, ""))
```
Comment (Scandie) [inline events/static/js/app/registration/form.js]: @BorysYudin change variable name `responce` to `response`

File: package.json [hunk 3]
```diff
@@ -23,11 +23,18 @@
...
     "react-router": "^3.0.0",
     "webpack": "^1.13.3",
-    "webpack-bundle-tracker": "0.0.93"
+    "webpack-bundle-tracker": "0.0.93",
+    "material-ui": "^0.16.1",
+    "react-tap-event-plugin": "^1.0.0",
+    "classnames": "^2.2.5",
+    "underscore": "^1.8.3",
+    "validator": "^6.1.0"
+  },
```
Comment (Scandie) [inline package.json:38]: @BorysYudin please, add axios to devDependencies

**Review by Scandie — COMMENTED** [review]
Code looks great, but you should fix some small issues.

**Review by Scandie — COMMENTED** [review]
Now it is OK. +1

File: events/registration/views.py [hunk 4]
```diff
@@ -27,12 +29,14 @@ def send_registration_confirm(user):
 
 class RegistrationView(View):
     def get(self, request):
-        data = {'first_name': '', 'last_name': '', 'email': '', 'password': '', 'birth_date': ''}
-        return JsonResponse(data, status=200)
```
Comment (yevgenbehen) [inline events/registration/views.py:31]: You do not need (). This is not "pythonic" style

**Comment by yevgenbehen** [comment]
+1

**Review by yevgenbehen — APPROVED** [review]

**Review by Scandie — APPROVED** [review]

**Pull request merged**